web-app/packages/lib/src/modules/dashboard/components/DashboardAccessRequestsRow.vue

```diff
@@ -6,54 +6,38 @@ SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-MerginMaps-Commercial
 
 <template>
   <app-container
-    v-if="currentNamespace && accessRequests && accessRequestsCount > 0"
+    v-if="
+      userStore.currentWorkspace &&
+      projectStore.accessRequests &&
+      projectStore.accessRequestsCount > 0
+    "
   >
     <app-section>
       <template #title
         >Requests
         <span class="text-color-medium-green"
-          >({{ accessRequestsCount }})</span
+          >({{ projectStore.accessRequestsCount }})</span
         ></template
       >
       <template #default>
-        <slot name="table" :namespace="currentNamespace"></slot>
+        <slot name="table" :namespace="userStore.currentWorkspace.name"></slot>
       </template>
     </app-section>
   </app-container>
 </template>
 
-<script lang="ts">
-import { mapActions, mapState } from 'pinia'
-import { defineComponent } from 'vue'
-
+<script lang="ts" setup>
 import { AppContainer, AppSection } from '@/common/components'
+import { useUserStore } from '@/main'
 import { useProjectStore } from '@/modules/project/store'
 
-export default defineComponent({
-  name: 'DashboardAccessRequestsRow',
-  components: { AppSection, AppContainer },
-  computed: {
-    ...mapState(useProjectStore, [
-      'accessRequests',
-      'accessRequestsCount',
-      'currentNamespace'
-    ])
-  },
-  watch: {
```

Comment:
on workspace switch (e.g. in EE) do we have reload of access requests?

Reply:
yes. setup is called on every mount of component. I tested it and everything was ok. Maybe @jozef-budac will some use case for such watching of workspace change.

Reply:
ok, thanks